dataset-spec.html

```diff
@@ -58,32 +58,19 @@
 <section>
   <h2>Data interfaces</h2>
 
-  <section data-dfn-for="Dataset">
-    <h3><dfn>Dataset</dfn> interface</h3>
+  <section data-dfn-for="CoreDataset">
+    <h3><dfn>CoreDataset</dfn> interface</h3>
```

Comment:
I have a slight preference for `DatasetCore` over `CoreDataset`. I perceive the first one as "the core of a dataset" and the 2nd one as "a dataset of core" (like a dataset of triples would a TripleDataset)?